src/paymentwall.php

```diff
@@ -226,6 +248,15 @@ function plgVmSetOnTablePluginParamsPayment ($name, $id, &$table)
     return $this->setOnTablePluginParams($name, $id, $table);
   }
 
+    /**
+     * Addition triggers for VM3
+     * @param $data
+     * @return bool
+     */
+    function plgVmDeclarePluginParamsPaymentVM3( &$data) {
```

Comment:
* Space: ok i will remove it.
* pass $data by reference: I follow this [tuts](http://docs.virtuemart.net/tutorials/development/175-code-adjustments-for-virtuemart-3.html). Is it important?